ostd/Cargo.toml

```diff
@@ -46,10 +46,9 @@ xarray = { git = "https://github.com/asterinas/xarray", version = "0.1.0" }
 [target.x86_64-unknown-none.dependencies]
 x86_64 = "0.14.2"
 x86 = "0.52.0"
-acpi = "4.1.1"
+acpi = { git = "https://github.com/rust-osdev/acpi.git", rev = "da9b3b1" }
```

Comment:
The problem is that OSTD can only depend on published crates.

Reply:
Changes have been released to `acpi` crate `v5.1.0`